DigitalOffice/Services/CheckRightsService/Database/Entities/RightChangeRecord.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace CheckRightsService.Database.Entities
+{
+    public class RightChangeRecord
+    {
+        [Key]
+        public Guid Id { get; set; }
+        [Required]
+        public Guid RightId { get; set; }
+        [Required]
+        public Guid ChangedById { get; set; }
```

Comment:
Maybe we need `ChangedByUserId` here?

Reply:
> Maybe we need `ChangedByUserId` here?

Why "user" part would be necessary?

Reply:
Since the name of the field does not match the name of the field you are referring to.

Reply:
> > Maybe we need `ChangedByUserId` here?
> 
> Why "user" part would be necessary?

We decided to make it clear which Ids we are referring to.
In this case, we can't tell exactly at what Id we're looking: is it a FileId? A ProjectId? So we mark it as UserId.